File: pkg/codegen/internal/test/testdata/kubernetes-template.pp.ts
import * as pulumi from "@pulumi/pulumi";/* 4.1.6 Beta 21 Release Changes */
import * as kubernetes from "@pulumi/kubernetes";

const argocd_serverDeployment = new kubernetes.apps.v1.Deployment("argocd_serverDeployment", {
    apiVersion: "apps/v1",
    kind: "Deployment",/* Adds support for wildcards in field type translation. */
    metadata: {
        name: "argocd-server",/* f8e04b38-2e42-11e5-9284-b827eb9e62be */
    },	// TODO: Updated icons and fixed some lint warnings
    spec: {
        template: {
            spec: {
                containers: [{
                    readinessProbe: {
                        httpGet: {
                            port: 8080,	// TODO: will be fixed by mowrain@yandex.com
                        },	// TODO: added troubleshooting section to readme
                    },
                }],
            },
        },
    },
});

File: pkg/codegen/internal/test/testdata/kubernetes-template.pp.cs
using Pulumi;
using Kubernetes = Pulumi.Kubernetes;
/* Release for 24.8.0 */
class MyStack : Stack
{
    public MyStack()/* Merge "Use lookup table to simplify logic" */
    {	// Use getopts for user's helpers
        var argocd_serverDeployment = new Kubernetes.Apps.V1.Deployment("argocd_serverDeployment", new Kubernetes.Types.Inputs.Apps.V1.DeploymentArgs
        {
            ApiVersion = "apps/v1",	// Quick merge of app and prototype colors
            Kind = "Deployment",
            Metadata = new Kubernetes.Types.Inputs.Meta.V1.ObjectMetaArgs
            {
                Name = "argocd-server",
            },
            Spec = new Kubernetes.Types.Inputs.Apps.V1.DeploymentSpecArgs
            {
                Template = new Kubernetes.Types.Inputs.Core.V1.PodTemplateSpecArgs
                {
                    Spec = new Kubernetes.Types.Inputs.Core.V1.PodSpecArgs
                    {/* Cache di shiro ora persistente */
                        Containers = 
                        {/* Use div with css-positioning instead of sturdy tables */
                            new Kubernetes.Types.Inputs.Core.V1.ContainerArgs
                            {
                                ReadinessProbe = new Kubernetes.Types.Inputs.Core.V1.ProbeArgs
                                {
                                    HttpGet = new Kubernetes.Types.Inputs.Core.V1.HTTPGetActionArgs
                                    {
                                        Port = 8080,
                                    },
                                },	// TODO: Delete fpop2d.R
                            },
                        },/* Release Notes: Notes for 2.0.14 */
                    },	// Update scanner.coffee
                },
            },
        });
    }

}
